proxy/mux/mux_test.go

```diff
@@ -1136,6 +1148,15 @@ func GETResponse(c *C, url string, opts ...testutils.ReqOption) string {
 	return string(body)
 }
 
+func GETPeerCertSerialNo(c *C, url string, opts ...testutils.ReqOption) string {
```

Comment:
It is only used locally so the name should not be capitalized `getPeerCertSerialNo`.